include/dca/parallel/mpi_concurrency/mpi_collective_sum.hpp

```diff
@@ -508,19 +567,97 @@ std::vector<Scalar> MPICollectiveSum::avgNormalizedMomenta(const func::function<
 }
 
 template <typename T>
-void MPICollectiveSum::sum(const T* in, T* out, std::size_t n) const {
-  // On summit large messages hangs if sizeof(floating point type) type * message_size > 2^31-1.
+void MPICollectiveSum::sum(const T* in, T* out, std::size_t n, int id) const {
```

Comment:
What is `int id` here?  There are several `id`s floating around this class.  Write function level doc here.

Reply:
it is documented in the declaration, line 167. Tough the name in the implementation did not match the declaration. Fixed in 7586540c6a2bd8e716e46595ff48c4b4f291fea3